cdap-proto/src/test/java/co/cask/cdap/proto/audit/AuditMessageTest.java

```diff
@@ -54,8 +54,8 @@
   @Test
   public void testCreateMessage() throws Exception {
     String dsCreateJson =
-      "{\"version\":1,\"time\":1000,\"entityId\":{\"namespace\":\"ns1\",\"dataset\":\"ds1\",\"entity\":\"DATASET\"}," +
-        "\"user\":\"user1\",\"type\":\"CREATE\",\"payload\":{}}";
+      "{\"version\":2,\"time\":1000,\"metadataEntity\":{\"details\":{\"namespace\":\"ns1\",\"dataset\":\"ds1\"}," +
```

Comment:
The changes in JSON is temporary to make the test pass. This will be reverted back to the original "entityId" when we will do the change to make it backward compatible.